arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+{
+	"swagger": "2.0",
```

Comment:
@rkmanda - You might want to check this project https://github.com/Azure/swashbuckle-resource-provider that shows you how to plugin swashbuckle to your webapi, that provides the swagger spec in the way we desire.

@yuzorma @yaqiyang would be the right people to ask questions for this tool if you are blocked.